### DISCUSSION TITLE
noImplicitOverride, override keyword

### DISCUSSION THREAD

**Discussion by Toby222** [vercel/next.js#25671] [Help]
I'm trying to start a new TypeScript/Next.js project (TS 4.3.2, next.js 10.2.3), for the first time with noImplicitOverride active, but it seems like compilation always fails, saying 'Syntax error: Unexpected token'
![image](https://user-images.githubusercontent.com/14962962/120298282-3e56f280-c2ca-11eb-82d0-8ca0d2077e2d.png)
Compiling with tsc isn't giving any problem when it runs and will gladly compile
```tsx
import React from "react";

export default class IndexPage extends React.Component {
  override render(): JSX.Element {
    return <span>Hello world</span>;
  }
}
```
down to 
```jsx
import React from "react";
export default class IndexPage extends React.Component {
    render() {
        return <span>Hello world</span>;
    }
}
```

Is there simply some option that I'm missing? Is this not supported yet (by some dependency or otherwise)?

**Comment by bentron2000** [comment]
+1 

I'm also having trouble when using `noImplicitOverride` in my tsconfig.

When set to `true` 
Including `override` will bork `next dev`, but build succeeds.
Without the `override`, `next dev` works, but build borks.

Something doesn't match 🤷‍♂️
